test/verify/check-static-login

```diff
@@ -714,27 +710,47 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         alice_cert_key = ['--cert', alice_cert, '--key', alice_key]
 
         # set up local (NIS) sssd provider and certificate mapping
-        m.write("/etc/sssd/sssd.conf", """
+        self.write_file("/etc/sssd/sssd.conf", """
 [sssd]
 domains = local
 
 [domain/local]
 id_provider = files
 
 [certmap/local/alice]
-# WARNING: interface testing only, this is insecure! Nothing validates the cert signer
+# WARNING: Requires sssd >= 2.6.1 and installing sssd_auth_ca_db.pem; with earlier sssd this is completely unsafe
 matchrule = <SUBJECT>^DC=LAN,DC=COCKPIT,CN=alice$
 """, perm="0600")
         m.execute("systemctl restart sssd")
 
-        # ensure sssd certificate lookup works
+        # ensure sssd certificate lookup without validation works
         user_obj = m.execute('busctl call org.freedesktop.sssd.infopipe /org/freedesktop/sssd/infopipe/Users '
                              'org.freedesktop.sssd.infopipe.Users FindByCertificate s -- '
                              '''"$(cat %s)" | sed 's/^o "//; s/"$//' ''' % alice_cert)
         self.assertEqual(m.execute('busctl get-property org.freedesktop.sssd.infopipe ' + user_obj.strip() +
                                    ' org.freedesktop.sssd.infopipe.Users.User name').strip(),
                          's "alice"')
 
+        # sssd 2.6.1 introduces FindByValidCertificate() method which validates the given certificate
+        # against the configured CA. Without this, certmap rules are completely unsafe!
+        api = m.execute("busctl introspect org.freedesktop.sssd.infopipe /org/freedesktop/sssd/infopipe/Users")
+        has_validate_api = 'FindByValidCertificate' in api
```

Comment:
I did this instead of our usual static OS list to avoid breaking packit tests and image refreshes, and forcing lockstep updates between bots and cockpit. However, let's make sure that it does not regress: Let's assert that this is True for fedora-35 and debian-testing. Then we can add more OSes as they come in, and we'll find that place when grepping for "fedora-35" (as we usually do with new OS releases).

Reply:
Done.